Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -54,6 +52,10 @@ namespace AZ
         // Node work lists using node count
         AZ_CVAR(uint32_t, r_numNodesPerCullingJob, 25, nullptr, AZ::ConsoleFunctorFlags::Null, "Controls amount of nodes to collect for jobs when not using the entry count");
 
+        //This value dictates the amount to extrude the octree node OBB when doing a frustum intersection test against the camera frustum to help cut draw calls. Default is set to 20 units as a reasonable value.
+        AZ_CVAR(int, r_shadowCascadeExtrusionAmount, 20, nullptr, AZ::ConsoleFunctorFlags::Null, "The amount to extrude the Obb towards light direction when doing frustum overlap test against camera frustum");
```

Comment:
Nit: Would be nice to say what the units are - are these meters?

Reply:
As discussed offline, perhaps consider replacing this distance with the projected distance between the cell and the camera.
![image](https://github.com/o3de/o3de/assets/82241079/bf600edd-abfc-400e-a70f-01dcf5c58ec1)
therefore
![image](https://github.com/o3de/o3de/assets/82241079/6ad5aaf1-55b3-49f9-b5bf-634b91f96ce0)

Reply:
This only works in some use cases. An example where it wont work is a following. By projecting onto light vector we get cy which does not extend to the frustum.
![IMG_1092](https://github.com/o3de/o3de/assets/47460854/2c3ed5d4-3877-4fcb-b5c9-334437ccc8df)

Reply:
It's guaranteed to intersect the camera if we applied the same algorithm to each point on the original bounding volume.

![image](https://github.com/o3de/o3de/assets/82241079/da09f0b3-5b5d-4859-bd1f-a4a5f80c7418)

Note all the points of intersection should lie on the B-D line segment. It seems to be close to a workable solution however, if we used the maximum segment length, say the distance between A-B and applied that to all points on the bounding volume to get the volume defined by A, B, C, E, we'd definitely get something that intersected the view frustum.

Reply:
> It's guaranteed to intersect the camera if we applied the same algorithm to each point on the original bounding volume.
> 
> ![image](https://user-images.githubusercontent.com/82241079/242988317-da09f0b3-5b5d-4859-bd1f-a4a5f80c7418.png)
> 
> Note all the points of intersection should lie on the B-D line segment. It seems to be close to a workable solution however, if we used the maximum segment length, say the distance between A-B and applied that to all points on the bounding volume to get the volume defined by A, B, C, E, we'd definitely get something that intersected the view frustum.

Directional lights are an orthographic projection and don't have an origin to work from. That means the calculation is then relative to the obb of the shadow cascade, which is not as cheap to calculate for every object potentially in the shadow casting frustum.

Reply:
Actually, I dont really need to get to point x which was going to be expensive. The dot product between ac and d will get me to  the camera and I will extend it a little further to get past the near plane and that should work. Its just a few vector ALUs so hopefully not significant on cpu. Ok let me try that approach.

Reply:
@kberg-amzn yes you are correct for that use case. I could in theory project the vector (centre of aabb to camera) onto light direction and it will tell me the distance from aabb to the camera and then I can extrude a little more to go past the near plane. But it break for other use cases. 

- Use case 1 is when the aabb is in front of the camera but still does not intersect the frustum as shown below.

![IMG_1154](https://github.com/o3de/o3de/assets/47460854/b4f4dbfb-46ce-40d5-af29-c89e3303b872)

- Use case 2 is when the aabb is behind the camera but further away such that even extending it to the calculated dot product may still miss the frustum unless you pad the frustum a little and then we are in the same boat as we are in right now where we dont know how much to pad. 

![IMG_1155](https://github.com/o3de/o3de/assets/47460854/7b2fc9aa-c7f1-4efa-afc6-c0a7d348d69d)

You could calculate the distance between center of aabb to the frustum by doing a vector to plane intersection for all the planes of the camera frustum and then extending the aabb to that distance. This will be expensive on cpu given that we calculate all the meshes of a cascade on one thread. It is not worth as this still doesnt help the worst case on gpu so in theory we are making the worst case even worse on cpu by doing all these extra tests but not rejecting any meshes. This solution also has a minor problem where some aabbs can eventually intersect with camera frustum under the ground and we will be incorrectly accepting them.